docs/devdoc/addDiffOp.md

```diff
@@ -0,0 +1,57 @@
+# Add a differential operator
+
+📜 _Solvers implemented in {py:mod}`qmat.solvers.generic` can be used_
+_with others {py:class}`DiffOp <qmat.solvers.generic.DiffOp>` classes_
+_than those implemented in {py:mod}`qmat.solvers.generic.diffops`._
+
+To add a new one, implement it at the end of the `diffops.py` module,
+using the following template :
+
+```python
+
+@registerDiffOp
+class Yoodlidoo(DiffOp):
+    r"""
+    Base description, in particular its equation :
+
+    .. math::
+
+        \frac{du}{dt} = ...
+
+    And some parameters description ...
+    """
+    def __init__(self, params="value"):
+        self.params = params
+        u0 = np.array([1, 0], dtype=float)
+        super().__init__(u0)
+
+    def evalF(self, u, t, out):
+        # TODO : your implementation
+        pass
+```
```

Comment:
I think this template is not entirely sufficient explanation. In particular because of some mixing of stuff. 
If I understand correctly, the "differential operator" implements the $f$ in an equation $u_t=f$. But then this class defines initial conditions for the associated initial value problem and has a function `evalF` when to me it should be called only `eval` because the differential operator is already $f$.
I have not yet read the code this documents, but that is also the intended order for new contributers and I am pretty sure I would not be the only one to be confused.